app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,298 @@
+            android:layout_marginStart="3dp"
+            android:src="@drawable/ic_star_icon"
+            app:layout_constraintStart_toEndOf="@id/first_star"
+            app:layout_constraintTop_toTopOf="@id/first_star" />
+
+        <ImageView
+            android:id="@+id/third_star"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="3dp"
+            android:src="@drawable/ic_star_icon"
+            app:layout_constraintStart_toEndOf="@id/second_star"
+            app:layout_constraintTop_toTopOf="@id/first_star" />
+
+        <ImageView
+            android:id="@+id/fourth_star"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="3dp"
+            android:src="@drawable/ic_star_icon"
+            app:layout_constraintStart_toEndOf="@id/third_star"
+            app:layout_constraintTop_toTopOf="@id/first_star" />
+
+        <ImageView
+            android:id="@+id/fifth_star"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="3dp"
+            android:layout_marginEnd="8dp"
+            android:src="@drawable/ic_star_icon"
+            app:layout_constraintStart_toEndOf="@id/fourth_star"
+            app:layout_constraintTop_toTopOf="@id/first_star"
+            app:layout_constraintEnd_toStartOf="@id/reviews"
+            app:tint="@color/tint_color" />
+
+        <TextView
+            android:id="@+id/reviews"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:fontFamily="@font/roboto_black"
+            android:text="@string/feedback_rate"
+            android:textColor="@color/tint_color"
+            android:textSize="14sp"
+            app:layout_constraintEnd_toStartOf="@id/guideline_end"
+            app:layout_constraintStart_toEndOf="@id/fifth_star"
+            app:layout_constraintTop_toBottomOf="@id/film_genre" />
+
+        <TextView
+            android:id="@+id/storyline"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="24dp"
+            android:fontFamily="@font/roboto_black"
+            android:text="@string/storyline"
+            android:textColor="@color/text_color"
+            android:textSize="14sp"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toStartOf="@id/guideline_end"
+            app:layout_constraintStart_toEndOf="@id/guideline_start"
+            app:layout_constraintTop_toBottomOf="@id/reviews" />
+
+        <TextView
+            android:id="@+id/description_text_view"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="6dp"
+            android:layout_marginBottom="6dp"
```

Comment:
remove
android:layout_marginBottom="6dp"

and add margin to cast_text_view if needed